dataland-e2etests/src/test/kotlin/org/dataland/e2etests/Constants.kt

```diff
@@ -41,6 +41,8 @@ val FRONTEND_DISPLAYED_FRAMEWORKS = listOf(
     DataTypeEnum.eutaxonomyMinusFinancials,
     DataTypeEnum.eutaxonomyMinusNonMinusFinancials,
     DataTypeEnum.lksg,
+    DataTypeEnum.sfdr,
+    DataTypeEnum.p2p,
```

Comment:
Why adding sfdr and p2p, but no sme?

Reply:
I think only datasets with view pages are listed here.

Reply:
ok makes sense